src/Sentry.Unity/SentryInitialization.cs

```diff
@@ -31,6 +32,12 @@ public static class SentryInitialization
         [RuntimeInitializeOnLoadMethod(RuntimeInitializeLoadType.BeforeSceneLoad)]
         public static void Init()
         {
+            if (!File.Exists($"{Application.dataPath}/Resources/{UnitySentryOptions.ConfigRootFolder}/{UnitySentryOptions.ConfigName}.json"))
+            {
+                Debug.LogWarning("Sentry Logging has been disabled.\nSentry has not been configured yet. Please go to Tools/Sentry...");
```

Comment:
If I understood correctly this will happen if the `SentryWindow` was never open, and therefore there was never a json file created.

Maybe we can mitigate this by creating the json file somehow.
Nonetheless we could reword the messageL
```suggestion
                Debug.LogDebug("Couldn't find the configuration file SentryOptions.json. Did you already configure Sentry?.\nYou can do that through the editor: Tools -> Sentry");
```

Reply:
Btw, forcing json creation sounds reasonable. We will be sure it's always created from our side. The logic about creation is inside `SentryWindow.OnEnable()`.

Reply:
We should look into creating it at package install time but we still need to do this validation to avoid a NullRefException if somehow the file isn't there. But in that case we could use `Debug.LogError` even to alert somethign went wrong.

Reply:
When you say "forcing" do you mean to force the user to open the window to create the json or that we somehow do that without user intervention?

Reply:
> We should look into creating it at package install time but we still need to do this validation to avoid a NullRefException if somehow the file isn't there. But in that case we could use `Debug.LogError` even to alert somethign went wrong.

I think we can hook into the AssetImportPipeline to get notified of the package gets installed/updated.

Reply:
We'll revisit the automated config-file generation in a separate issue. https://github.com/getsentry/sentry-unity/issues/136#issue-871121703